patterns/.stylelintrc

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
it might be faster to understand the meaning of the "patterns" folder, it is renamed as "frontend". what do you think?

Reply:
I'm agree with you, done